add 3-2

FedZik opened the pull request:
![image](https://github.com/FedZik/TKI-142/assets/145327286/cb4efc0b-283f-4a4a-9885-2294c0ea40ad)

FedZik committed:
add 3-2

BalakinaEP commented on task_3-2/3-2.c:
```diff
@@ -0,0 +1,67 @@
+ 
+double getSum(int count) 
+{ 
+        double current = 0.006944444444444;
```
не подходит
нужно, чтобы функция считала значение

BalakinaEP reviewed (changes requested):
пункт б)

FedZik committed:
fix

FedZik committed:
update

удаление ранее не замеченных строк
добавление второго пункта задания
удаление лишних библиотек

MarinaPaley commented on task_3-2/3-2.c:
```diff
@@ -0,0 +1,78 @@
+ 
+double getSum(int count) 
+{ 
+        double current = findFirst();
```
`getRecurrent(0);`

MarinaPaley commented on task_3-2/3-2b.c:
```diff
@@ -0,0 +1,81 @@
+    double sum = current; 
+    for (int k = startPoint; k < endPoint; k++) 
+    { 
+            current *= getRecurrent(k); 
```
`while (fabs(current) > epsilon){}`

FedZik committed:
rework

delete 3-2b

MarinaPaley commented on task_3-2/3-2.c:
```diff
@@ -0,0 +1,111 @@
+	double current = getReccurent(0);
+	double sum = current;
+	int k = 1;
+	while (current >= e)
```
`+ DBL_EPSILON`
